frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -314,7 +358,15 @@ export const surveyLogic = kea<surveyLogicType>([
             {},
             {
                 loadSurveyRatingResultsSuccess: (state, { payload }) => {
-                    return { ...state, [`question_${payload?.questionIndex}`]: true }
+                    return { ...state, [`${payload?.questionIndex}`]: true }
+                },
+            },
+        ],
+        surveySingleChoiceResultsReady: [
+            {},
+            {
+                loadSurveySingleChoiceResultsSuccess: (state, { payload }) => {
+                    return { ...state, [`${payload?.questionIndex}`]: true }
```

Comment:
I think we need some error handling here since that should be the only case when questionIndex doesn't exist in the payload and this incorrectly adds 'true' to undefined.

(and a more general error state for when the hogql query fails, vanishing / infinite loading the graph is suboptimal) - but ok to do in a follow up